README.md

```diff
@@ -29,13 +59,15 @@ We support **ASP.NET Core** application as well as **.NET Core** application.
 Customize configurations are supported for `v1.0.2+`. There are several ways to customize the settings. For example:

```

Comment:
It is not clear what the version number refers to. Suggest something like

Custom configuration are supported starting with version 1.0.2 of the `ApplicationInsights-Kubernetes` package.